home-manager/gnome.nix

```diff
@@ -248,6 +248,11 @@ in
         autoconnect = [ "qemu:///system" ];
         uris = [ "qemu:///system" ];
       };
+
+      # https://fcitx-im.org/wiki/Using_Fcitx_5_on_Wayland
+      "org.gnome.settings-daemon.plugins.xsettings" = {
```

Comment:
typo https://github.com/kachick/dotfiles/commit/e3bf507a35a19d1499011294253fcb74e6dabe4e

Reply:
Removed in https://github.com/kachick/dotfiles/commit/3605181e40b4c680450fd94244738bd1bdcde3c3